### ISSUE TITLE
Have the text property automatically filled when the react property is used when sending emails.

### ISSUE THREAD

**Issue by ValentinH** [resend/resend-node#369]
Do you recommend to always send a text version of an email along with the html version? If yes, could we have the SDK automatically set the `text` property when we use the `react` one (doing a second render with the `plainText: true` option)?
- https://github.com/resend/resend-node/blob/canary/src/emails/emails.ts#L30
- https://github.com/resend/resend-node/blob/canary/src/batch/batch.ts#L27

**Comment by bukinoshita** [comment]
> Do you recommend to always send a text version of an email along with the html version? If yes, could we have the SDK automatically set the `text` property when we use the `react` one (doing a second render with the `plainText: true` option)?
> 
> * https://github.com/resend/resend-node/blob/canary/src/emails/emails.ts#L30
> * https://github.com/resend/resend-node/blob/canary/src/batch/batch.ts#L27

Yes, having the text version of the email is usually recommended.
That's a great suggestion, ideally this should be on API side so user never needs to send the text version doesn't matter which SDK they are using. Doing on SDK side with React Email is easier though

We will explore more

**Comment by jessethomson** [comment]
It seems this still hasn't been implemented.

In the meantime, is this how you all would recommend working around this?
```jsx
import { Resend } from 'resend';
import { render } from '@react-email/render';

const resend = new Resend('re_123456789');

async function sendEmail() {
    const jsx = <MyTemplate />;
    const text = await render(jsx, {
        plainText: true,
    });

    await resend.emails.send({
        from: 'Acme <onboarding@resend.dev',
        to: ['delivered@resend.dev'],
        subject: 'hello world',
        react: jsx,
        text,
    });
}
```

**Comment by gabrielmfern** [comment]
Closing as a duplicate of #451

**Comment by isabellaaquino** [comment]
We just released a fix for this on our API side! We will autogenerate the plain text version if not included. Thanks for reporting!

**Comment by sommeeeer** [comment]
Thank you @isabellaaquino !
